packages/orbit-components/src/Slider/components/Histogram/index.tsx

```diff
@@ -74,25 +19,35 @@ const Histogram = ({ data, value, min, loading = false, loadingText, step }: Pro
   const highlightFrom = Array.isArray(value) ? value[0] : 0;
   const highlightTo = Array.isArray(value) ? value[value.length - 1] : value;
   return (
-    <StyledHistogram role="presentation">
+    <div
+      className="h-xxxl flex items-end justify-start gap-px overflow-hidden pb-[3px]"
+      role="presentation"
+    >
       {loading ? (
-        <StyledLoadingContainer>
+        <div className="flex size-full items-center justify-center">
           <Loading type="inlineLoader" text={loadingText} />
-        </StyledLoadingContainer>
+        </div>
       ) : (
         data &&
         data.map((column, index) => {
           const properIndex = (index + min) * step;
+          const active = properIndex >= highlightFrom && properIndex <= highlightTo;
+          const height = maxValue ? `${((column / maxValue) * 100).toFixed(2)}%` : 0;
           return (
-            <StyledHistogramColumn
+            <div
               key={encodeURIComponent(properIndex.toString())}
-              height={maxValue && +((column / maxValue) * 100).toFixed(1)}
-              active={properIndex >= highlightFrom && properIndex <= highlightTo}
-            />
+              className={cx(
+                "relative flex-auto",
+                active ? "bg-blue-normal" : "bg-blue-light-active",
+              )}
+              style={{ height }}
+            >
+              <div className="absolute bottom-[-3px] h-px w-full bg-inherit" />
```

Comment:
It's still missing here. The `rounded-[1px]` is both in the histogram column and the `:after`.

![image](https://github.com/kiwicom/orbit/assets/8863238/5b541b21-5568-4670-a40c-90b791a71d31)

Reply:
For me it doesn’t even render anything since the height is `1px` but added it back.

![Screenshot 2024-02-15 at 13 16 43](https://github.com/kiwicom/orbit/assets/821851/d217fbf5-78e3-4590-8b4c-26f222e67968)

Reply:
Interesting.. ok let's leave it like that. Sorry for the trouble.

Reply:
No prob at all. Thank you for the review 🙂